src/App.js

```diff
@@ -1,11 +1,57 @@
 import "./App.css";
+import Header from "./components/Header.js";
+import ToDoList from "./components/ToDoList.js";
+import { useState } from "react";
+
+// const toDoArray = [
+//   { name: "shoppingItem", status: "Pending" },
+//   { name: "running", status: "Done" },
+// ];
+
+export default function App() {
+  const [toDo, setToDo] = useState([]);
+
+  function handleAddToDo(toDoItem) {
+    const newToDo = [
```

Comment:
Same here, i'd use `newToDos` in plural, because it's holding an array of objects.